airflow/pipe_events_encounters_dag.py

```diff
@@ -25,6 +25,8 @@ def build(self, dag_id):
 
         with DAG(dag_id, schedule_interval=self.schedule_interval, default_args=self.default_args) as dag:
             self.config = config
+            publish_to_postgres = '{publish_to_postgres}'.format(**config) == 'True'
```

Comment:
@smpiano this shouldn't be necesary, you can do:
`publish_to_postgres = config.get('publish_to_postgres',false)`